Fix HybridAppLoginTimeOut documentation (#6874)
mendix/docs: content/en/docs/refguide/runtime/custom-settings/_index.md

| <a id="EnableKeepAlive" href="#EnableKeepAlive">EnableKeepAlive</a> | Defines whether the web client sends a keep alive request every SessionTimeout/2 milliseconds, to prevent a session timeout. Each click in the browser also acts as KeepAlive. Disabling this property will result in a user being logged out automatically after 10 minutes of inactivity, even if the browser remains open. | true |
| <a id="PhoneUserAgentRegEx" href="#PhoneUserAgentRegEx">PhoneUserAgentRegEx</a> | Defines the regular expression that is used to determine whether a user is visiting a Mendix application from a phone. The regular expression is matched against the user-agent header sent by the client's web browser. | Android, Mobile (iPhone, iPod, BlackBerry) |
| <a id="TabletUserAgentRegEx" href="#TabletUserAgentRegEx">TabletUserAgentRegEx</a> | Defines the regular expression that is used to determine whether a user is visiting a Mendix application from a tablet. The regular expression is matched against the User-Agent header sent by the client's web browser. | Android, iPad |

> | <a id="commendixwebuiHybridAppLoginTimeOut" href="#commendixwebuiHybridAppLoginTimeOut">com.<wbr>mendix.<wbr>webui.<wbr>HybridAppLoginTimeOut</a> | Determines how many minutes your token will remain valid before re-authenticating using your full credentials. By default the token will never expire. | |

Reviewer: @atennapel 
Hi Albert,
So if you set this it has to have a value for how many minutes before the token expires, but if you don't specify this setting at all it will never expire?

Author: Yes, that is correct.

Reviewer: I think the idea of a 'default' could be confusing. Can I change this to
```suggestion
| <a id="commendixwebuiHybridAppLoginTimeOut" href="#commendixwebuiHybridAppLoginTimeOut">com.<wbr>mendix.<wbr>webui.<wbr>HybridAppLoginTimeOut</a> | Determines how many minutes your token will remain valid before re-authenticating using your full credentials. If no value is set the token will never expire. | |
```
?

Author: Yes better!
